docs/framework/wpf/app-development/build-and-deploy-how-to-topics.md

```diff
@@ -24,6 +24,6 @@ The following topics show how to create project files for the various [!INCLUDE[
   
  [Deploying a WPF Application](../../../../docs/framework/wpf/app-development/deploying-a-wpf-application-wpf.md)  
   
- [How to: Create a New WPF Application Project](https://msdn.microsoft.com/library/1f6aea7a-33e1-4d3f-8555-1daa42e95d82)  
+ [How to: Create a New WPF Application Project](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/bb546958(v=vs.100))  
```

Comment:
let's remove these just previous-versions links and instead link to https://docs.microsoft.com/en-us/dotnet/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application

Reply:
@mairaw, did you want to use that new link to replace `[How to: Create a New WPF Browser Application Project](https://msdn.microsoft.com/library/72ef4d90-e163-42a1-8df0-ea7ccfd1901f)` as well?

Reply:
Nope. It's just in this case.